.github/workflows/stale.yml

```diff
@@ -0,0 +1,30 @@
+# https://github.com/actions/stale#usage
+name: Stale pull requests
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Midnight.
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v8
+        with:
+          days-before-pr-stale:    365
+          days-before-pr-close:    30
+          days-before-issue-stale: 1825  # 365 * 5
+          days-before-issue-close: 90
```

Comment:
No issues please

Reply:
Can we instead use a huge value like `365 * 10` ?. 🤔